test/fixtures/test-runner/output/test-runner-plan.js

```diff
@@ -1,7 +1,36 @@
 'use strict';
-const { test } = require('node:test');
+const { test, suite } = require('node:test');
 const { Readable } = require('node:stream');
 
+suite('input validation', () => {
```

Comment:
I would actually add all of these new tests to a separate "regular" test file. I don't think we necessarily need to check snapshot output for these. Snapshot tests are more tedious to maintain over time (they need regenerated and introduce merge conflicts). Plus, the linter doesn't run on fixture files.

Reply:
As soon as possible I'll update these tests